addon/components/rdfa/rdfa-editor.ts

```diff
@@ -116,7 +119,7 @@ export default class RdfaEditor extends Component<RdfaEditorArgs> {
     if (this.args.rdfaEditorInit) {
       this.args.rdfaEditorInit(rdfaDocument);
     }
-    this.initializeComponents();
+    // this.initializeComponents();
```

Comment:
Are we keeping this commented for any reason

Reply:
No my mistake, I will remove it.